src/server/apis/settings.ts

```diff
@@ -8,71 +8,32 @@ import {OrgDetailsQuery} from '../../types/gql-types';
 function getRouter(github: GitHub): express.Router {
   const settingsRouter = express.Router();
   settingsRouter.post('/orgs.json', async (request: express.Request, response: express.Response) => {
-    try {
-      const loginDetails = await getLoginFromRequest(github, request);
-      if (!loginDetails) {
-          response.sendStatus(400);
-          return;
-      }
+    const loginDetails = await getLoginFromRequest(github, request);
+    if (!loginDetails) {
+        response.sendStatus(400);
+        return;
+    }
 
+    try {
       // TODO: Run github query to get org web hook state etc
-      let orgDetails;
-      try {
-        orgDetails = await github.query<OrgDetailsQuery>({
-          query: orgsDetailsQuery,
-          fetchPolicy: 'network-only',
-          context: {
-            token: loginDetails.token,
-          }
-        });
-      } catch (err) {
-        // TODO: Hack until scopes can be pushed.
-        orgDetails = {
-          data: {
-            viewer: {
-              organizations: {
-                nodes: [
-                  {
-                    name: 'Google',
-                    viewerCanAdminister: false
-                  },
-                  {
-                    name: 'Udacity',
-                    viewerCanAdminister: false
-                  },
-                  {
-                    name: 'HTML5Rocks',
-                    viewerCanAdminister: false
-                  },
-                  {
-                    name: 'PolymerLabs',
-                    viewerCanAdminister: false
-                  },
-                  {
-                    name: 'Web Starter Kit',
-                    viewerCanAdminister: true
-                  },
-                  {
-                    name: 'GoogleChromeLabs',
-                    viewerCanAdminister: true
-                  }
-                ]
-              }
-            }
-          }
-        }
-      }
 
-      const orgs = orgDetails.data.viewer.organizations.nodes;
+      const orgDetails = await github.query<OrgDetailsQuery>({
```

Comment:
We should perhaps store the granted scopes by token when the user logs in. Maybe file an issue for this? Or a TODO somewhere?

Reply:
There is the `X-OAuth-Scopes` but I can't see a clean place at the moment to pull it from the login / graphql stuff.

Github Info: https://developer.github.com/apps/building-oauth-apps/scopes-for-oauth-apps/